src/poster/poster.js

```diff
@@ -26,6 +26,16 @@ function Poster({ slide, content, run, slideDone, executionId }) {
   const [show, setShow] = useState(true);
   const timerRef = useRef(null);
   const animationTimerRef = useRef(null);
+  const logo = slide?.themeData?.logo;
+  const { showLogo } = content;
+  const logoStyle = {};
+
+  let logoUrl = "";
+  // If showlogo is set, get the logo url
+  if (logo && showLogo) {
+    logoUrl = getFirstMediaUrlFromField(slide.mediaData, logo);
+    logoStyle.backgroundImage = `url("${logoUrl}")`;
```

Comment:
Why insert it as a background image, and not in an img tag?

Reply:
I have no philosophy behind this choice, would you prefer <img?

Reply:
Just thought of it more as a piece of content rather than a decorative background. And i think we might get easier control over size that way.

Reply:
I'll change it